packages/block-library/src/video/tranforms.js

```diff
@@ -28,8 +28,8 @@ const transforms = {
 			attributes: {
 				src: {
 					type: 'string',
-					shortcode: ( { named: { src } } ) => {
-						return src;
+					shortcode: ( { named: { src, mp4, m4v, webm, ogv, wmv, flv } } ) => {
+						return src || mp4 || m4v || webm || ogv || wmv || flv;
```

Comment:
Hey @jorgefilipecosta 

Just looking at the code for the video shortcode, and I'm not sure wmv is supported:
https://github.com/WordPress/wordpress-develop/blob/d36eda33f72f7a4fd2cfb693a84c6c1fd985cb56/src/wp-includes/media.php#L2610

I wasn't able to find any uses of the filter in that function. Might be worth double checking that one.

Reply:
Nice catch @talldan 👍 The code was updated.